corehq/apps/es/client.py

```diff
@@ -445,6 +450,14 @@ def reindex(
         }
         if purge_ids:
             reindex_body["script"] = {"inline": "if (ctx._source._id) {ctx._source.remove('_id')}"}
+            if script_json:
+                raise ValueError("You can't use both purge_ids and script_json at once")
+        elif script_json:
+            with open(script_json) as f:
+                reindex_body["script"] = json.loads(f.read())
+        if source_query_json:
+            with open(source_query_json) as f:
+                reindex_body["source"]["query"] = json.loads(f.read())
```

Comment:
We will be able to run below command to reindex unknownusers from old index to new index.

```
/manage.py elastic_sync_multiplexed start users --script_json=corehq/apps/es/management/commands/scripts/add_domain_memberships.json --source_query_json=corehq/apps/es/management/commands/scripts/query_unknown_users.json
```